### PR TITLE
chore: move discord support to a dedicated repository

### PR DESCRIPTION
**Pull request by Rushaway**
+dependabot support

https://github.com/srcdslab/sm-plugin-KnifeAlert-discord

### CODE REVIEW

File: addons/sourcemod/scripting/include/KnifeAlert.inc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * Called right after an Human knife a Zombie
+ *
+ * @param attacker          Attacker UserId
+ * @param Auth_attacker     Attacker SteamID  
+ * @param victim            Victim UserId
+ * @param pOldKnifer        Initial knifer UserId
+ * @param Auth_OldKnifer    Initial knifer SteamID 
+ * @noreturn
+ */
+forward void KnifeAlert_OnInfection(int attacker, char[] Auth_attacker, int victim, int pOldKnifer, char[] Auth_OldKnifer);
```
Comment (maximee-leroyy) [inline addons/sourcemod/scripting/include/KnifeAlert.inc:26]: we can use the zombiereloaded forwards no ?

File: addons/sourcemod/scripting/include/KnifeAlert.inc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+/**
+ * Called right after an Human knife a Zombie
```
Comment (maximee-leroyy) [inline addons/sourcemod/scripting/include/KnifeAlert.inc:29]: whats the purpose of this forward

File: addons/sourcemod/scripting/include/KnifeAlert.inc (hunk 1 above)
Comment (Rushaway) [reply addons/sourcemod/scripting/include/KnifeAlert.inc:26]: Zombie Reloaded is optional, plugin can be used for others game mod.

Comment (maximee-leroyy) [reply addons/sourcemod/scripting/include/KnifeAlert.inc:26]: good point, so maybe use another word because it might not be related to zr

**Pull request closed without merging**